bin/cli/src/lib/write-ui-env-file.ts

```diff
@@ -34,8 +35,8 @@ export async function writeUiEnvFile(stage, local = false) {
     VITE_COGNITO_USER_POOL_CLIENT_ID: deploymentOutput.userPoolClientId,
     VITE_COGNITO_USER_POOL_CLIENT_DOMAIN: deploymentOutput.userPoolClientDomain,
     VITE_COGNITO_REDIRECT_SIGNIN: local
-      ? `"http://localhost:5000/"`
-      : deploymentOutput.applicationEndpointUrl,
+      ? `"http://localhost:5000/dashboard"`
+      : `${deploymentOutput.applicationEndpointUrl}dashboard`,
```

Comment:
Does this need a `/` here?

Reply:
Nope the applicationEndpointUrl ends with "/"